sky/services/semantics/semantics.mojom

```diff
@@ -0,0 +1,69 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module semantics;
+
+struct SemanticsNode {
+  uint32 id;
+  SemanticFlags? flags;
+  SemanticStrings? strings;
+  SemanticGeometry? geometry;
+  array<SemanticsNode> children;
+};
+
+struct SemanticFlags {
```

Comment:
The use of the plural form here indicates that the values must be used to form a mask.  Shouldn't this be something like:
````
struct SemanticFlags {
  canBeTapped =                1 << 0,
  canBeLongPressed =           1 << 1,
  canBeScrolledHorizontally =  1 << 2,
  canBeScrolledVertically =    1 << 3,
  hasCheckedState =            1 << 4,
  isChecked =                  1 << 5,
}
```
Unless my understanding of struct definitions in mojoms is completely off.

OTOH, if you don't want to create a mask, the `SemanticsNode` should probably have an array of these (`array<SemanticFlags>`). For example, something being both horizontally and vertically scrollable (array or mask, probably the latter given your comment on the packing)